Fabulous.CodeGen/src/Fabulous.CodeGen/AssemblyReader/Models.fs

```diff
@@ -59,6 +59,9 @@ module Models =
           /// The extracted properties
           Properties: AssemblyTypeProperty array
           
+          /// The constructor parameters to be used to instantiate the type
+          ConstructorsParameters: string[][]
```

Comment:
You might want to make a new type instead of just `string` here, to add `uniqueName` to avoid name collision.
```
"ContructorParameters: [
    {
        "name": "androidContext",
        "type": "Android.Context"
    },
    {
        "name": "MyAwesomeContext",
        "type": "AwesomeLib.Context"
    }
]"
```

Reply:
Also I'm not about the 2-dimension array. Only 1 constructor will be used.

Reply:
> You might want to make a new type instead of just `string` here, to add `uniqueName` to avoid name collision.

Sure

> Also I'm not sure about the 2-dimension array. Only 1 constructor will be used.

We can not access to `ConstructorParameters` value  on `ReadAssembly` step at this moment. So, I just store all constructors in two dimensional array and then try to find constructor based on `ConstructorParameters` value on `Binder` step (see my comments).